source/sass/components/article-page.scss

```diff
@@ -56,18 +50,22 @@
 }
 
 .article-summary-link {
+  display: block;
   --spacing: 10px;
 
   margin-top: 17px;
   flex-basis: calc(50% - var(--spacing));
   flex-grow: 0;
   flex-shrink: 0;
-
   @include hover-focus-active {
     color: inherit;
   }
 }
 
+.article-summary-link-two-column {
```

Comment:
if there are more than 5 items, the article summary links are then given the "Article-summary-link-two-column" class which makes them inline block, to appear with uniform spacing between them